cms/src/main/java/uk/nhs/digital/apispecs/swagger/request/examplerenderer/CodegenParameterExampleHtmlRenderer.java

```diff
@@ -101,7 +102,10 @@ private String htmlFrom(final Collection<ParamExample> complexExamplesFromParamS
     }
 
     private String htmlFrom(final String simpleExampleValue) {
-        return MessageFormat.format("Example: <span class=\"nhsd-a-text-highlight nhsd-a-text-highlight--code\">{0}</span>", simpleExampleValue);
+        return MessageFormat.format(
```

Comment:
any reason why MessageFormat.format is used instead of just String.format?

Reply:
Purely personal preference as I find `MessageFormat`'s syntax of using indexed `{0}` placeholders more intuitive and flexible.

Hopefully, specifying `MessageFormat` explicitly makes it clear enough that this doesn't take `String.format`-compliant pattern?

TBH, I wish the API went even further and supported named placeholders but we can't have it all, I suppose :grin:.

Reply:
i just wish string interpolation/string manipulation in general was as simple as in javascript! :laughing: